embedded-pubsub/pom.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>testcontainers-spring-boot</artifactId>
+        <groupId>com.playtika.testcontainers</groupId>
+        <version>1.31-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>embedded-pubsub</artifactId>
```

Comment:
embedded-googlecloud-pubsub? embedded-google-pubsub?
just pubsub is too generic, there is redis pub-sub and others...

Reply:
@renanreismartins could you please rename artifact and folder?